tests/tt_eager/python_api_testing/unit_testing/misc/test_moreh_matmul.py

```diff
@@ -229,6 +236,8 @@ def test_moreh_matmul_backward(params, input_b1, input_b2, other_b1, other_b2, r
         logger.debug(f"other_grad passing={passing}")
         logger.debug(f"other_grad pcc={output_pcc}")
         assert passing
+    else:
+        assert tt_other_grad is None
 
 
 @skip_for_wormhole_b0()
```

Comment:
I would appreciate it if you could remove `@skip_for_wormhole_b0()` from all tests.

Reply:
> I would appreciate it if you could remove `@skip_for_wormhole_b0()` from all tests.

I removed all ```@skip_for_wormhole_b0()```

Reply:
Thanks for this example related to #5044